### ISSUE TITLE
Docs: music is way to loud on introduction video

### ISSUE THREAD

**Issue by SebastianThorn** [grafana/grafana#57670]
**Steps to reproduce**:
* Go to https://grafana.com/docs/grafana/latest/administration/service-accounts/
* Press play on the video

**Actual Result**:
* Music is louder then the person talking.

**Expected Result**
* To properly hear the person talking.

**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)

**Environment**:
- Grafana version: N/A
- Data source type & version: N/A
- User OS & Browser: N/A
- Others: N/A

**Comment by GrafanaWriter** [comment]
I'm unable to replicate this (MacOS Version 12.4, running Chrome).

@SebastianThorn - Can you share the operation system and browser you were using?

**Comment by SebastianThorn** [comment]
If you don't think it's too loud, leave it as is.
